AcademyChallenge/Data/Service/LiveReposStorage.swift

```diff
@@ -12,12 +12,9 @@ import CoreData
 //
 import RxSwift
 class LiveReposStorage: ReposService {
-    
     private	var reposNetwork: Network = .init()
     // private let persistence: ReposCoreData = .init()
-    
-    func fetchRepos(page: Int, size: Int) -> Single<[Repos]> {
-        
-        return reposNetwork.rxExecuteNetworkCall(ReposAPI.getRepos(perPage: size, page: page))
+    func fetchRepos(page: Int, size: Int) -> Single<[Repos]> {        
+        return reposNetwork.rx.rxExecuteNetworkCall(ReposAPI.getRepos(perPage: size, page: page))
```

Comment:
the `rx` in `rxExecuteNetworkCall` can be removed

Reply:
solved